src/Sylius/Behat/spec/Context/Ui/UserContextSpec.php

```diff
@@ -42,4 +52,54 @@ function it_logs_in_user_with_given_credentials($loginPage)
 
         $this->iLogInAs('john.doe@example.com', 'password123');
     }
+
+    function it_deletes_account(
+        $userRepository,
+        $customerShowPage,
+        $sharedStorage,
+        UserInterface $user,
+        CustomerInterface $customer
+    ) {
+        $userRepository->findOneByEmail('theodore@test.com')->shouldBeCalled()->willReturn($user);
+        $sharedStorage->setCurrentResource('deletedUser', $user)->shouldBeCalled();
+
+        $user ->getCustomer()->shouldBeCalled()->willReturn($customer);
```

Comment:
And again, I think we agreed, that ``shouldBeCalled()->willReturn(...)`` are redundant everywhere except builders.